pakyow-presenter/lib/presenter/view.rb

```diff
@@ -336,6 +336,15 @@ def component_name
 
       private
 
+      def adjust_value_parts(value, parts)
+        return value unless value.respond_to?(:to_hash)
+
+        parts_to_keep = parts.fetch(:include) { value.keys } 
```

Comment:
Let's specify the default as the second argument to `fetch`. Applies to this line and the one after.